### PR TITLE
Update tex packages

### PR DESCRIPTION
**Pull request by zorkow**
This is now ready for review:

* Adds docs for new extension packages
* Adds summary of implemented macros for all other existing packages (except for `textmacros` which already had an extensive list).
* Updates the index of all implemented macros
* Updates the configuration file `conf.py` to work with newer sphinx processors.

### CODE REVIEW

File: input/tex/extensions/upgreek.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+upgreek
+#######
+
+The `upgreek` extension implements the ``upgreek`` style package from LaTeX. It
+provides upright Greek characters for both lower and upper cases.  See the `CTAN
+page <https://www.ctan.org/pkg/upgreek>`__ for more information and
+documentation of `upgreek`.
+
+Note, that the extension does not implement the font selection options from the
+LaTeX package.
```
Comment (dpvc) [inline input/tex/extensions/upgreek.rst:13]: Missing "This package isn't autoloaded" message.

**Review by dpvc — APPROVED** [review]
A few minor details, but otherwise looks good.

File: input/tex/extensions/ams.rst [hunk 2]
```diff
@@ -38,4 +38,21 @@ remove it:
      tex: {packages: {'[-]': ['ams']}}
    };
 
+
+-----
```
Comment (dpvc) [inline input/tex/extensions/ams.rst:42]: There are now options for the ams extension.  Can those be added here?

File: input/tex/extensions/colortbl.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+   cell of a row to color. If used elsewhere an error is thrown.
+
+.. describe:: \columncolor[model]{color}
+
+   Allows to color a single column. It needs to be used in the first **cell** of
+   the column to color. If used elsewhere an error is thrown.
+
+   Note, that it is unlike its LaTeX counterpart that is used in the layout
+   specification of a tabular environment. While MathJax will not throw an error
+   of a ``\columncolor`` is used in the arguments of the tabular environment, it
```
[the remark this answers is not in the document]
Comment (zorkow) [reply input/tex/extensions/colortbl.rst:27]: I see the confusion. That was not intended to refer to the `tabular` environment at all. Replacing `tabular` with `table` plus clarifying example.

File: input/tex/extensions/gensymb.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+gensymb
+#######
+
+The `gensymb` extension implements the ``gensymb`` style package from LaTeX. It
+provides a number of macros for unit notation.  See the `CTAN page
+<https://www.ctan.org/pkg/gensymb>`__ for more information and documentation of
+`gensymb`.
+
+Note that not all the characters for this package are yet included in the
+MathJax fonts so output might vary on clients.
```
[the remark this answers is not in the document]
Comment (zorkow) [reply input/tex/extensions/gensymb.rst:13]: Added.

File: input/tex/extensions/upgreek.rst (hunk 1 above)
Comment (zorkow) [reply input/tex/extensions/upgreek.rst:13]: Added.

File: input/tex/extensions/centernot.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+.. _tex-centernot-commands:
+
+
+centernot Commands
+------------------
+
+The `centernot` extension implements the following macros:
+``\centernot``, ``\centerOver``
```
[the remark this answers is not in the document]
Comment (zorkow) [reply input/tex/extensions/centernot.rst:57]: I've added the description.

File: input/tex/extensions/ams.rst (hunk 2 above)
Comment (zorkow) [reply input/tex/extensions/ams.rst:42]: Added the descriptions.

**Comment by zorkow** [comment]
I wrote docu for some of the packages before I integrated the `(not) autoload` paragraph into the template generator of `tex_packages.js`. But I think I caught now all packages where the paragraph was missing.

**Pull request merged**